File: src/Java/Fibonacci/algorithm.java
public class algorithm {
    public static void main(String[] args) {
        int num,
            num1 = 0,
            num2 = 1,
            temp;
        num = Integer.parseInt(args[0]);

        System.out.println("Fibonacci :");

        for(int i = 1; i <= num; i++) {
            System.out.println(num1 + " ");
            temp = num1 + num2;
            num1 = num2;
            num2 = temp;
        }
    }
}

File: src/C++/Fibonacci/algorithm.cpp
#include <stdio.h>
#include <iostream>
using namespace std;

int i, v, num1 = 0, num2 = 1, temp;

int main() {
    cout<<"Enter the number of Fibonacci numbers you want :"<<endl;
    scanf("%d", &v);
    printf("Fibonacci nums : ");

    for(i; i <= v; i++) {
        temp = num1 + num2;
        num1 = num2;
        num2 = temp;
        cout<<num1<<" ,";
    }

    return 0;
}